shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -931,19 +935,27 @@ - (void)dispatchTouches:(NSSet*)touches
   [_engine.get() dispatchPointerDataPacket:std::move(packet)];
 }
 
-- (void)touchesBegan:(NSSet*)touches withEvent:(UIEvent*)event {
+- (void)flutterView:(FlutterView*)flutterView
```

Comment:
Are `flutterView` and `event` necessary?

Reply:
That's idiomatic objc, when you have a delegate the first parameter should be the sender.  The event is as necessary as anywhere else events are handled =)